packages/react-components/react-menu/src/components/MenuTrigger/MenuTrigger.types.ts

```diff
@@ -20,9 +20,13 @@ export type MenuTriggerChildProps<Type extends ARIAButtonType = ARIAButtonType,
     'aria-expanded'?: boolean;
     id: string;
     ref: React.Ref<never>;
+    // eslint-disable-next-line @fluentui/consistent-callback-type
```

Comment:
suggestion: for multiple disables it might be better idea to use eslint:disable / eslint:enable multi line pragma